python/BUILD

```diff
@@ -60,6 +76,12 @@ checkstyle_test(
         "WORKSPACE",
         "requirements*.txt",
         "typedb/**/*",
+        "tool/sphinx/*",
+    ]),
+    exclude = glob([
+        "conf.py",
```

Comment:
We didn't end up calling these `sphinx_conf.py` and `sphinx_index.rst`?

Reply:
I left an answer for that comment: https://github.com/vaticle/typedb-driver/pull/450#discussion_r1352919629